docs/website/docs/tutorial/filesystem.md

```diff
@@ -0,0 +1,276 @@
+
+## 1. Setting up a new project
+
+To help you get started quickly, `dlt` provides some handy CLI commands. One of these commands will help you set up a new `dlt` project:
+
+```
+dlt init filesystem duckdb
+```
+
+This command creates a project that loads data from a filesystem into a DuckDB database. You can easily switch out duckdb for any other [supported destinations](../dlt-ecosystem/destinations).
+After running this command, your project will have the following structure:
+
+```txt
+filesystem_pipeline.py
+requirements.txt
+.dlt/
+    config.toml
+    secrets.toml
+```
+
+Here’s what each file does:
+
+- `filesystem_pipeline.py`: This is the main script where you'll define your data pipeline. It contains several different examples of loading data from filesystem source.
+- `requirements.txt`: This file lists all the Python dependencies required for your project.
+- `.dlt/`: This directory contains the [configuration files](../general-usage/credentials/) for your project:
+    - `secrets.toml`: This file stores your API keys, tokens, and other sensitive information.
+    - `config.toml`: This file contains the configuration settings for your dlt project.
+
+:::note
+When deploying your pipeline in a production environment, managing all configurations with files
+might not be convenient. In this case, you can use environment variables or other [configuration providers](../general-usage/credentials/setup#available-config-providers) available in
+`dlt` to store secrets and configs instead.
+:::
+
+## 2. Creating the pipeline
+
+Filesystem source provides users with building blocks for loading data from any type of files. You can break down the data extraction into two steps:
+
+1. Accessing files in the bucket / directory.
+2. Reading the files and yielding records.
+
+`dlt`'s filesystem source includes several resources that you can use together or individually:
+
+- `filesystem` resource accesses files in the directory or bucket
+- several readers resources (`read_csv`, `read_parquet`, `read_jsonl`) read files and yield the records
+
+Let's initialize a source and create a pipeline for loading parquet files from an AWS S3 to DuckDB:
+
+```
+import dlt
+from dlt.sources.filesystem import filesystem, read_parquet
+
+parquet_resource = filesystem(file_glob="**/*.parquet") | read_parquet()
+pipeline = dlt.pipeline(pipeline_name="s3_to_duckdb", dataset_name="parquet_data", destination="duckdb")
+info = pipeline.run(parquet_resource)
+print(info)
+```
+
+You can replace the example code from `filesystem_pipeline.py` with the code above. What's happening in this snippet?
+
+1. We initialize the filesystem resource and pass the `file_glob` parameter. Based in this parameter, `dlt` will filter
+all files in the bucket.
+2. We pipe the transformer `read_parquet` to read the files yielded by filesystem resource and iterate over records
```

Comment:
Would suggest to rephrase to something like: We use a pipe to pass files yielded into the `read_parquet` transformer, which will read the file and iterate over its records.